.github/workflows/test.yml

```
@@ -0,0 +1,23 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [8, 10, 12]
```

```suggestion
        node: [14]
```

Not sure why we were testing 8, 10 and 12. We don't need that here because `npm test` only lints, so LTS will suffice.